packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -71,39 +66,76 @@ test.describe('When modal is closed', () => {
                         isDismissible: true,
                     },
                     on: {
-                        click: (event: string) => messages.push(event),
+                        [ON_MODAL_CLOSE_EVENT]: (event: Event) => events.push(event),
                     },
                 },
             );
 
-            await page.locator(closeButtonSelector).click();
+            await page.click(closeButtonSelector);
 
             // Assert
-            expect(messages).toHaveLength(1);
+            expect(events).toHaveLength(1);
+        });
+
+        test('should close the modal', async ({ mount, page }) => {
+            // Arrange
+            await mount(PieModal, {
+                props: {
+                    isOpen: true,
+                    isDismissible: true,
+                },
+            });
+
+            const modal = page.locator(modalSelector);
+            expect(modal).toBeVisible();
```

Comment:
Do you need this expect?

Reply:
About to have a chat with Josh about this, likely not, although it does prove the test works! 😄